src/System.Runtime/tests/System/Int64.cs

```diff
@@ -38,261 +36,230 @@ public static void TestMinValue()
     }
 
     [Theory]
-    [InlineData((long)234, 0)]
-    [InlineData(long.MinValue, 1)]
-    [InlineData((long)(-123), 1)]
-    [InlineData((long)0, 1)]
-    [InlineData((long)45, 1)]
-    [InlineData((long)123, 1)]
-    [InlineData((long)456, -1)]
-    [InlineData(long.MaxValue, -1)]
-    public static void TestCompareTo(long value, long expected)
-    {
-        long i = 234;
-        long result = CompareHelper.NormalizeCompare(i.CompareTo(value));
-        Assert.Equal(expected, result);
-    }
-
-    [Theory]
-    [InlineData(null, 1)]
-    [InlineData((long)234, 0)]
-    [InlineData(long.MinValue, 1)]
-    [InlineData((long)(-123), 1)]
-    [InlineData((long)0, 1)]
-    [InlineData((long)45, 1)]
-    [InlineData((long)123, 1)]
-    [InlineData((long)456, -1)]
-    [InlineData(long.MaxValue, -1)]
-    public static void TestCompareToObject(object obj, long expected)
+    [InlineData((long)234, (long)234, 0)]
+    [InlineData((long)234, long.MinValue, 1)]
+    [InlineData((long)234, (long)-123, 1)]
+    [InlineData((long)234, (long)0, 1)]
+    [InlineData((long)234, (long)123, 1)]
+    [InlineData((long)234, (long)456, -1)]
+    [InlineData((long)234, long.MaxValue, -1)]
+    [InlineData((long)234, null, 1)]
+    public static void TestCompareTo(long i, object value, long expected)
```

Comment:
We did right?
`[InlineData((long)234, (long)-123, 1)]`

Or do you mean comparing a negative long to a positive one or a more negative one?